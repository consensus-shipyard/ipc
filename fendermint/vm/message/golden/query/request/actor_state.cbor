a16a4163746f725374617465550100c44d3544d7fef86bfa4ec69096050100b70f11