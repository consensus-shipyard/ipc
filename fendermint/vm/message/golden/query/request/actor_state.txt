ActorState(Address { payload: Secp256k1([0, 196, 77, 53, 68, 215, 254, 248, 107, 250, 78, 198, 144, 150, 5, 1, 0, 183, 15, 17]) })